fh-ios-sdk/Categories/JSON/FHJSON.m

```diff
@@ -1,9 +1,19 @@
-//
-//  FHJSON.m
-//  fh-ios-sdk
-//
-//  Copyright (c) 2012-2015 FeedHenry. All rights reserved.
-//
+/*
+ * JBoss, Home of Professional Open Source.
```

Comment:
@matzew I reused the one we used for AeroGear so we're consistent across both projects.